html/apps/tweetfeed/services/dataService.js

```diff
@@ -0,0 +1,44 @@
+(function(){
+    angular.module('myApp')
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["angular" is not defined.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2614812995/issues/source?bid=3288751&fileBranchId=3312487#l2)